apps/api/openapi/openapi.json

```diff
@@ -787,6 +690,114 @@
           "500": {
             "description": "Server error"
           }
+        },
+        "parameters": [
```

Comment:
Not sure how this is working it's way into the spec, but we shouldn't be adding the bearer token header like this.  Instead we should be using the BearerAuth security directive.

e.g.
```
"security": [
  {
     "BearerAuth": []
  }
],
```
 Here are docs describing this idea https://swagger.io/docs/specification/v3_0/authentication/bearer-authentication/

Reply:
It looks like there are two duplicate routes being defined in `account.routes.ts`.  Must be a rebase or merge conflict resolution error.  

Checkout line 393 and 553
https://github.com/recallnet/js-recall/blob/main/apps/api/src/routes/account.routes.ts#L393
https://github.com/recallnet/js-recall/blob/main/apps/api/src/routes/account.routes.ts#L553